app.json

```diff
@@ -211,6 +211,10 @@
       "description": "The OAuth client secret configured in the edX instance",
       "required": true
     },
+    "ENABLE_CERTIFICATE_DASHBAORD_LINKS": {
```

Comment:
Since this a feature flag that will be short-lived, can you name it FEATURE_ENABLE_CERTIFICATE_DASHBAORD_LINKS ?

Also, does it make sense to use one feature flag for both the dashboard links and the user view #1103 ?

Reply:
Yeah sure, I will refactor that variable name.

As far as the usage of `FEATURE_ENABLE_CERTIFICATE_DASHBAORD_LINKS` is concerned, I don't exactly know the difference between dashboard-links and user view, are not both the same? 
According to the current implementation, if this flag is turned off, user will not be able to view his certificate link/button on the dashboard/application page.